CODEOWNERS

```diff
@@ -16,7 +16,7 @@ src/include/user/**			@thesofproject/steering-committee
 src/include/sof/debug/gdb/*		@mrajwa
```

Comment:
You miss typed the last part of the address:

`WARNING: Missing Signed-off-by: line by nominal patch author 'Marcin Rajwa <marcin.rajwa@linux.intel.com>`

Reply:
@mrajwa As a tip to avoid this in the future, you can just do "git commit -s" and that will automatically put the correct Signed-off-by: line.

Reply:
which will also make sure the sign off email matches the author email